next.clayui.com/gatsby/createPages.js

```diff
@@ -71,6 +72,8 @@ const createDocs = (actions, edges, mdx) => {
 
```

Comment:
If adding `sibling` to the `.md` files, we can skip creating the `.mdx` pages here.